runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -262,6 +263,10 @@ public static DataflowRunner fromOptions(PipelineOptions options) {
         "DataflowRunner requires stagingLocation, and it is missing in PipelineOptions.");
     validator.validateOutputFilePrefixSupported(dataflowOptions.getStagingLocation());
 
+    if (!Strings.isNullOrEmpty(dataflowOptions.getSaveProfilesToGcs())) {
```

Comment:
is there an option to disable validation?

(conceptually, the user might not have access, while the VMs do have)